api/src/main/java/org/openmrs/module/fhirExtension/export/impl/ProcedureFormExport.java

```diff
@@ -0,0 +1,223 @@
+package org.openmrs.module.fhirExtension.export.impl;
+
+import lombok.extern.log4j.Log4j2;
+import org.apache.commons.lang3.StringUtils;
+import org.hl7.fhir.instance.model.api.IBaseResource;
+import org.hl7.fhir.r4.model.Annotation;
+import org.hl7.fhir.r4.model.CodeableConcept;
+import org.hl7.fhir.r4.model.DateTimeType;
+import org.hl7.fhir.r4.model.MarkdownType;
+import org.hl7.fhir.r4.model.Procedure;
+import org.openmrs.Concept;
+import org.openmrs.Obs;
+import org.openmrs.api.AdministrationService;
+import org.openmrs.api.ConceptService;
+import org.openmrs.api.ObsService;
+import org.openmrs.module.fhir2.api.translators.ConceptTranslator;
+import org.openmrs.module.fhirExtension.export.Exporter;
+import org.openmrs.util.OpenmrsUtil;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+import java.util.concurrent.atomic.AtomicReference;
+
+@Component
+@Log4j2
+public class ProcedureFormExport implements Exporter {
+
+	private final AdministrationService adminService;
+	
+	private final ConceptTranslator conceptTranslator;
+	
+	private final ConceptService conceptService;
+	
+	private final ObsService obsService;
+
+	private static final String PROCEDURE_TEMPLATE_PROPERTIES_FILE_NAME = "procedure-template.properties";
```

Comment:
in ABDM India distro, they already have a mapping file. Is that a different file name? and is the format same?

Reply:
@angshu file name is different (abdm_config.properties) and format is same

Reply:
@angshu made changes to read procedure record template file name from global property.